src/v3/src/hooks/useValue.ts

```diff
@@ -16,7 +16,7 @@ import { FieldElement } from '../types';
 export const useValue = (uischema: FieldElement): unknown => {
   const { name } = uischema.options.inputMeta;
 
-  const { data } = useWidgetContext();
+  const { data, additionalData } = useWidgetContext();
 
-  return data[name];
+  return { ...additionalData, ...data }[name];
```

Comment:
What is `additionalData` and why isn't it in `data`?

Reply:
Additional data (right now) really just contains the `confirmPassword` data, it was recommended to create this state prop for fields that should not be submitted to auth-js. I added it here as to not repeat the useValue logic just for this field. It wont cause cause because data and additional data should always contain unique keys.